src/components/ListItem/ListItem.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './ListItem.css';
+import Item from '../../components/Item/Item';
+var Columns = require('react-columns');
+function ListItem() {
+  var queries = [{
+    columns: 2,
+    query: 'min-width: 500px'
+  }, {
+    columns: 3,
+    query: 'min-width: 1000px'
+  }];
+  return (
+    <Columns queries={queries}>
+      <Item />
```

Comment:
Add items into list and render each item in the list

something like:
```
const items = [
   Item,
   Item,
   ...
]

```
and at the render
```
   <Columns quaries={quaries}>
     { items.map((item, i) => ( <Item key={i} /> ) }
   </Columns>
```